R/clear.cache.R

```diff
@@ -65,3 +65,8 @@ clear.cache <- function (...){
         }
 }
 
+# Determine if a variable is cached or not
+.is.cached <- function (variable) {
+        file.exists(file.path(.cache.dir, paste0(variable, ".RData")))
```

Comment:
Is this guaranteed to test the same condition as `.read.cache.info` does it, and do you think that the `in.cache` test in `.read.cache.info` should call this function instead of implementing two ways to check if the variable is in cache? They should do exactly the same and there's no better guarantee than coding it in one place only.
Also, I'm thinking this function should really be in `cache.R`, or all cache related auxiliary functions should be placed together in a single file.